src/applications/gqrx/mainwindow.cpp

```diff
@@ -668,8 +670,8 @@ void MainWindow::updateFrequencyRange(bool ignore_limits)
         qDebug() << QString("New frequnecy range: %1 - %2 MHz (step is %3 Hz but we use 1 Hz).").
                     arg(startd*1.0e-6).arg(stopd*1.0e-6).arg(stepd);
 
-        qint64 start = (qint64)startd + d_lnb_lo;
-        qint64 stop  = (qint64)stopd  + d_lnb_lo;
+        qint64 start = (qint64)(startd + rx->get_filter_offset()) + d_lnb_lo;
+        qint64 stop  = (qint64)(stopd + rx->get_filter_offset())  + d_lnb_lo;
```

Comment:
I came to think about the same thing after I posted my comment but I'm still not happy with this solution.

It feels simply wrong that every time we change the filter offset, which could be 10-20 times per second, we call updateFrequencyRange(), which in turn calls rx->get_rf_range(), which calls an external API call that we have no control over. Besides, it always returns the same limit, so it is waste of CPU. On the other hand I realize that the same argument can be used against lnb_lo :-(

At the very least we should update the comment "This function must also be called when the LNB LO has changed." to include the filter offset.